src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -3778,6 +3778,30 @@ pub const TCP_BBR_USEDEL_RATE: c_int = 1079;
 pub const TCP_BBR_MIN_RTO: c_int = 1080;
 pub const TCP_BBR_MAX_RTO: c_int = 1081;
 pub const TCP_BBR_ALGORITHM: c_int = 1083;
+pub const TCP_BBR_PACE_PER_SEC: c_int = 1086;
+pub const TCP_BBR_PACE_DEL_TAR: c_int = 1087;
+pub const TCP_BBR_PACE_SEG_MAX: c_int = 1088;
+pub const TCP_BBR_PACE_SEG_MIN: c_int = 1089;
+pub const TCP_BBR_PACE_CROSS: c_int = 1090;
+pub const TCP_RACK_TLP_USE: c_int = 1095;
+pub const TCP_BBR_TMR_PACE_OH: c_int = 1096;
+pub const TCP_RACK_DO_DETECTION: c_int = 1097;
+pub const TCP_BBR_RACK_RTT_USE: c_int = 1098;
+pub const TCP_BBR_RETRAN_WTSO: c_int = 1099;
+pub const TCP_DATA_AFTER_CLOSE: c_int = 1100;
+pub const TCP_BBR_PROBE_RTT_GAIN: c_int = 1101;
+pub const TCP_BBR_PROBE_RTT_LEN: c_int = 1102;
+pub const TCP_BBR_SEND_IWND_IN_TSO: c_int = 1103;
+pub const TCP_BBR_USE_RACK_RR: c_int = 1104;
+pub const TCP_BBR_USE_RACK_CHEAT: c_int = TCP_BBR_USE_RACK_RR;
```

Comment:
We probably only need one of these. It looks like `TCP_BBR_USE_RACK_RR` was added more recently https://github.com/freebsd/freebsd-src/commit/e570d231f48957dcf0757b7b716330f3bd64e362#diff-c354fa9fe15a3e5c90079f38792a8e5458d76677437a53ee7537a62fbc72e100R255.